docs/security.md

```diff
@@ -1,20 +1,25 @@
-# Security Policy
+# freeCodeCamp.org's Security Policy
 
 This document outlines our security policy for the codebases, platforms that we operate, and how to report vulnerabilities.
 
 ## Reporting a Vulnerability
 
-If you think you have found a vulnerability, _please report responsibly_. Don't create GitHub issues for security issues. Instead, please send an email to `security@freecodecamp.org` and we'll look into it immediately.
+> [!NOTE]
+> If you think you have found a vulnerability, **please report responsibly**. Do not create GitHub issues for security issues. Instead follow this guide.
 
-Ensure that you are using the **latest**, **stable** and **updated** version of the Operating System and Web Browser available to you on your machine.
+### Guidelines
 
-We appreciate any responsible disclosure of vulnerabilities that might impact the integrity of our platforms and users.
+We appreciate a responsible disclosure of vulnerabilities that might impact the integrity of our platforms and users. In the interest of saving everyone time, we encourage you to report vulnerabilities with these in mind:
```

Comment:
```suggestion
We appreciate responsible disclosure of vulnerabilities that might impact the integrity of our platforms and users. In the interest of saving everyone's time, we encourage you to report vulnerabilities with these in mind:
```
Bit late, I know, but the 'a' doesn't belong here. Also `'s` is missing

Reply:
Not sure about the `'s`.